docs/requirements.md

```diff
@@ -1,16 +1,59 @@
-# PerfCheck CI Requirements
+# Perf Check CI Requirements
 
 Perf Check CI is a web application to manage jobs that measure and compare performance on another web application.
 
 ## Creating a job
 
-* As a user I want to create a job to compare the performance of two branches across multiple request paths.
+* As a user, I want to compare relative performance of my branch against master to make sure I'm not shipping code that will slow things down.
+* As a user, I want to have a quick idea how performance on my branch looks, without caring about comparisons against master.
+* As a user, I want to create a job to compare the performance of two branches across multiple request paths.
+* As a user, I know the request I'm working on takes a long time (10-60 seconds), so just want a very rough comparison that takes a smaller amount of time to complete so I can get back to work.
+* As a user, I care about getting a final and accurate-as-possible comparison across two branches.
```

Comment:
Is this story about trust?

* As a user, I want Perf Check CI to return final, stable, and accurate comparisons between two branches so everyone involved in the project can understand and trust the result.

Reply:
This is a way of specifying that people might want to increase ‘n’ for increased accuracy, for example after arriving at a final result after optimization.